core/transaction_pool/transaction_pool.hpp

```diff
@@ -12,6 +12,8 @@
 
 namespace kagome::transaction_pool {
 
+  using primitives::Transaction;
```

Comment:
In transaction_pool we include both this header and pool_moderator. So we have 2 using for the same thing. I wonder how it is not considered as error, but we should not have this. 
I don't see a big problem writing primitives::Transaction everytime